src/main/java/com/spotinst/sdkjava/example/oceanCD/OceanCDUsageExample.java

```diff
@@ -329,5 +380,232 @@ private static Boolean rolloutAction(OceanCDClient client, String rolloutId) {
         return updateStatus;
     }
 
+    private static String createVerificationProvider(OceanCDClient client) {
+        System.out.println("-------------------------start creating Ocean CD VerificationProvider------------------------");
+
+        List<String> clusterIds = Arrays.asList("Cluster1","Cluster2");
+
+        //Build VerificationProviderCloudWatch
+        VerificationProviderCloudWatch.Builder verificationProviderCloudWatchBuilder = VerificationProviderCloudWatch.Builder.get();
+        VerificationProviderCloudWatch verificationProviderCloudWatch = verificationProviderCloudWatchBuilder.setIamArn("arn:aws:iam::253244684816:role/Automation-Test").build();
+
+        //Build VerificationProvider
+        VerificationProvider.Builder VerificationProviderBuilder = VerificationProvider.Builder.get();
+        VerificationProvider VerificationProvider = VerificationProviderBuilder.setName("Test-VerificationProvider").setCloudWatch(verificationProviderCloudWatch).setClusterIds(clusterIds).build();
+
+
+        VerificationProvider createdVerificationProvider = client.createVerificationProvider(VerificationProvider);
+        System.out.println("VerificationProvider successfully created: " + createdVerificationProvider.getName());
+
+        return createdVerificationProvider.getName();
+
+    }
+
+    private static VerificationProvider getVerificationProvider (OceanCDClient client, String VerificationProviderName) {
+
+        System.out.println(String.format("Get Ocean CD VerificationProvider. VerificationProvider Name: %s", VerificationProviderName));
+
+        return client.getVerificationProvider(VerificationProviderName);
+    }
+
+    private static List<VerificationProvider> getAllVerificationProviders (OceanCDClient client) {
+
+        System.out.println("Get All Ocean CD VerificationProviders");
+        return client.getAllVerificationProviders();
+    }
+
+    private static Boolean updateVerificationProvider(OceanCDClient client, String VerificationProviderName) {
+        System.out.println("-------------------------start updating Ocean CD VerificationProvider------------------------");
+
+        List<String> clusterIds = Arrays.asList("Cluster1","Cluster2");
+
+        //Build VerificationProviderNewRelic
+        VerificationProviderNewRelic.Builder verificationProviderNewRelicBuilder = VerificationProviderNewRelic.Builder.get();
+        VerificationProviderNewRelic verificationProviderNewRelic = verificationProviderNewRelicBuilder.setAccountId("act-b114530s").setPersonalApiKey("b634e07cca045cec80f16eb7").build();
```

Comment:
Remove key and account id from here. Keep it global with <account-id> <api-key> values and reference here.

Reply:
updated